src/main/webapp/js/gameInfo.js

```diff
@@ -1,61 +1,107 @@
+/**
+* Loads the game data to the game info page
+*/
 function loadGameData() {
   var gameID = getGameID();
   const params = new URLSearchParams();
   params.append('gameID', gameID)
   var request = new Request('/load-gamepage-data', {method: 'POST', body: params});
   fetch(request).then(response => response.json()).then((data) => {
-    printMapName(data);
+    printMapNameAndDifficulty(data);
     printMapDescription(data);
     printMapImage(data);
     printMapRating(data);
     printMapCreator(data);
     printPlayGameButton(data);
-    printRateMapButton(data);
-    printMapDifficulty(data);
-    printMapComments(data);
+    // printRateMapButton(data);
+    // printMapComments(data);
   });
 }
 
-function printMapName(data) {
+/**
+* Adds the game name and difficulty where the div with id 'map-name' is
+* @param {string} data is the JSON from the server ‘/load-gamepage-data’ 
+*/
+function printMapNameAndDifficulty(data) {
   var mapName = document.getElementById('map-name');
-  mapName.innerHTML = data.gameName;
+  var difficultyClass = 'red-text';
+  var difficulty = '[Hard]'
+  if (data.difficulty == 1) {
+    difficultyClass = 'green-text';
+    difficulty = '[Easy]';
+  } else if ( data.difficulty == 2) {
+    difficultyClass = 'orange-text';
+    difficulty = '[Medium]';
+  }
+  mapName.innerHTML = '<h1>' + data.gameName + ' <i class=' + difficultyClass +'>' + difficulty + '</i></h1>';
 }
 
+/**
+* Adds the game description where the div with id 'map-description' is
+* @param {string} data is the JSON from the server ‘/load-gamepage-data’ 
+*/
 function printMapDescription(data) {
   var mapDescription = document.getElementById('map-description');
-  mapDescription.innerHTML = data.gameDescription;
+  mapDescription.innerHTML = '<h5>Game Description: </h5>';
+  mapDescription.innerHTML += '<h4>' + data.gameDescription + '</h4>';
 }
 
+/**
+* Adds the game image where the div with id 'map-image' is
+* @param {string} data is the JSON from the server ‘/load-gamepage-data’ 
+*/
 function printMapImage(data) {
   var mapImage = document.getElementById('map-image');
-  mapImage.innerHTML = data.stageLocations;
+  mapImage.append(createStaticMap(data.stageLocations, 500, data.gameID));
 }
 
+/**
+* Adds the game rating where the div with id 'map-rating' is
+* @param {string} data is the JSON from the server ‘/load-gamepage-data’ 
+*/
 function printMapRating(data) {
   var mapRating = document.getElementById('map-rating');
-  mapRating.innerHTML = data.stars;
+  mapRating.innerHTML = getStarRating(data.stars).replace('md-18', 'large').replace('md-18', 'large').replace('md-18', 'large').replace('md-18', 'large').replace('md-18', 'large');
```

Comment:
Use the regex /md-18/g instead of 'md-18' to replace all instances of this string.
`getStarRating(data.stars).replace(/md-18/g, 'large')`

Reply:
Oh woah didn't know that would do the trick, thank you and done!